src/api/accessToken.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint @typescript-eslint/camelcase: 0 */
+import express from 'express'
+import uuidv4 from 'uuid/v4'
+import { AccessToken } from '../types/general'
+import { setData, getAccessTokens } from '../lib/data'
+
+const accessTokensRouter = express.Router()
+
+accessTokensRouter.post('/', (req, res) => {
```

Comment:
could we do a check here? if `req.body.name` length is < 2, then its an invalid name (too short or empty), send back 400.